### PR TITLE
Admins must now complete math problems to admin

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

See title.
image

## Why It's Good For The Game

We need to ensure that the admins on the server are fully mentally prepared to administrate. As such we can ensure this by giving them math problems that they need to solve.

## Changelog

:cl:
admin: Admins are now required to complete math problems before being allowed to admin.
/:cl:

### CODE REVIEW

**Review by jlsnow301 — APPROVED** [review]
it's about time someone takes this issue seriously

**Comment by Higgin** [comment]
it's over

**Comment by Draggeru** [comment]
I just spent 500 dollars on plushies, I don't think I'm cut out for adminning anymore

**Comment by atlasle** [comment]
which tramstation train arrives first

File: code/modules/admin/admin_math.dm [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+/proc/do_admin_math(client/client)
+	var/ckey = client.ckey
+
+	var/static/list/in_math_problem = list()
+	if(ckey in in_math_problem)
+		return FALSE
+	in_math_problem += ckey
+
+	var/static/list/failure_timers = list()
+	if(ckey in failure_timers)
```
Comment (LemonInTheDark) [inline code/modules/admin/admin_math.dm:39]: it is more efficent to do [] then in

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Review by ZephyrTFA — COMMENTED** [review]
turns out admins can't do math

**Pull request closed without merging**